packages/errors/src/codes.ts

```diff
@@ -96,6 +96,7 @@ export const SOLANA_ERROR__KEYS__INVALID_KEY_PAIR_BYTE_LENGTH = 3704000 as const
 export const SOLANA_ERROR__KEYS__INVALID_PRIVATE_KEY_BYTE_LENGTH = 3704001 as const;
 export const SOLANA_ERROR__KEYS__INVALID_SIGNATURE_BYTE_LENGTH = 3704002 as const;
 export const SOLANA_ERROR__KEYS__SIGNATURE_STRING_LENGTH_OUT_OF_RANGE = 3704003 as const;
+export const SOLANA_ERROR__KEYS__VERIFY_SIGNATURE_FAILURE = 3704004 as const;
```

Comment:
To be more consistent with the other errors this should be `SOLANA_ERROR__KEYS__FAILED_TO_VERIFY_SIGNATURE`.

However, I think we should instead call this error something like `SOLANA_ERROR__KEYS__PUBLIC_KEY_MUST_MATCH_PRIVATE_KEY` because what we're actually trying to test here is that the private and public keys match. The fact that we are verifying a signature is secondary.